src/bridge/command.rs

```diff
@@ -31,24 +31,29 @@ fn set_windows_creation_flags(cmd: &mut TokioCommand) {
 }
 
 fn build_nvim_cmd() -> TokioCommand {
-    if let Some(path) = SETTINGS.get::<CmdLineSettings>().neovim_bin {
-        // if neovim_bin contains a path separator, then try to launch it directly
-        // otherwise use which to find the fully path
-        if path.contains('/') || path.contains('\\') {
-            if neovim_ok(&path) {
-                return build_nvim_cmd_with_args(&path);
-            }
-        } else if let Some(path) = platform_which(&path) {
-            if neovim_ok(&path) {
-                return build_nvim_cmd_with_args(&path);
+    if let Some(cmdline) = SETTINGS.get::<CmdLineSettings>().neovim_bin {
+        if let Some((bin, args)) = shlex::split(&cmdline)
+            .filter(|tokens| !tokens.is_empty())
+            .map(|mut tokens| (tokens.remove(0), tokens))
```

Comment:
I changed to split the tokens into the first argument and the rest within this method to reduce the indent. As a bonus, I could remove one redundant clone comparing to my previous implementation since now it modifies the `Vec` instance directly.

Reply:
Nice, yes this looks good to me.